bootrom

```diff
@@ -0,0 +1 @@
+src/main/resources/testchipip/bootrom/
```

Comment:
Is this a symlink now? Is it still necessary to have this?

Reply:
They aren't necessary for my project-template PR, but I don't know how everyone else uses testchipip. I thought it best to make the least intrusive change.